lib/aws/amplify/api/user.ts

```diff
@@ -0,0 +1,54 @@
+import { extractTheSoulOfTheUser } from "../../../auth/sessionToken.ts"
+import { API_URL } from "../../../api/endpoints/users.ts";
+
+export async function checkUserInDynamoDB() {
```

Comment:
client shouldn't know about the backend implementations and the name should be generic like getUserStatus or any other name without the DB name